server/services/statementServiceTypes.ts

```diff
@@ -0,0 +1,18 @@
+type UserStatement = {
+  additionalComments?: Array<{ additionalComment: string; dateSubmitted: Date }>
+  id?: number
+  bookingId?: number
+  incidentDate?: Date
+  lastTrainingMonth?: number
+  lastTrainingYear?: number
+  jobStartYear?: number
+  statement?: string
+  submittedDate?: Date
+  name?: string
+  reporterName?: string
+}
+
+type Status = {
```

Comment:
as I said there is a real type that could be surfaced. We need to change the types in `server/config/types.js` to enums and then we could explicitly refer to `StatementStatus` as a type in its own right. 

This is fine for now though and better than leaving untyped! I often have made mistakes with these fake javascript enums

Reply:
ok